src/testoptix/stringtable.cpp

```diff
@@ -0,0 +1,126 @@
+#include "stringtable.h"
```

Comment:
Let's put the usual license boilerplate here.

Reply:
This can be fixed later, but if we think StringTable is going to be useful to all renderers using OSL+OptiX, at some point we should probably move it from testoptix to liboslexec.